app/src/poi.js

```diff
@@ -31,6 +31,30 @@ export default function Poi() {
         },
       });
 
+      const popup = new mapboxgl.Popup({
+        closeButton: false,
+        closeOnClick: false,
+      });
+      map.current.on("mousemove", "points-of-interest", (e) => {
+        // Change the cursor to a pointer when the mouse is move over the points-of-interest layer.
+        map.current.getCanvas().style.cursor = "pointer";
+        // Copy coordinates array.
+        const coordinates = e.features[0].geometry.coordinates.slice();
+        const address_data = e.features[0].properties;
```

Comment:
```suggestion
        const locationData = e.features[0].properties;
```
In JS we use `camelCase`